valispace/__init__.py

```diff
@@ -100,8 +100,8 @@ def all_vali_names(self):
 
 		return valinames.json()
 
-	def get_vali(self, id=None, name=None):
-		""" Returns the correct Vali. Input can be id or name """
+	def get_vali(self, id=None, name=None, workspace_id=None, project_id=None, parent_id=None, parent_name=None, tag_id=None, tag_name=None, valis_marked_as_impacted=None):
+		""" Returns the correct Vali(s) """
```

Comment:
Another thing to indicate in the comment here is that if you pass the id or name, all the other parameters will be ignored. And if you don't pass the id or name, the other parameters can be combined to form a filter.

Reply:
In general perhaps it would be better to have a separate get and filter function, with the get function always returning a single Vali and the filter a list of results (similar to Django objects.get and objects.filter).